src/lib/containers/home_page/featured-data/FacetPlotsCard.tsx

```diff
@@ -107,59 +121,65 @@ const FacetPlotsCard: React.FunctionComponent<FacetPlotsCardProps> = ({
     if (detailsPagePath && selectedFacetValue) {
-      detailsPageLink = <div className="FacetPlotsCard__body__footer">
-        <div className="FacetPlotsCard__body__footer__link">
-          <a href={detailsPagePath}>
-            View {selectedFacetValue}
-          </a>
+      detailsPageLink = (
+        <div className="FacetPlotsCard__body__footer">
+          <div className="FacetPlotsCard__body__footer__link">
+            <a href={detailsPagePath}>View {selectedFacetValue}</a>
+          </div>
         </div>
-      </div>
+      )
     }
     const isShowingMultiplePlots = facetPlotDataArray.length > 1
-    const cardTitle = title ?? (isShowingMultiplePlots ? selectedFacetValue : unCamelCase(facetDataArray[0].columnName, facetAliases))
+    const cardTitle =
+      title ??
+      (isShowingMultiplePlots
+        ? selectedFacetValue
+        : unCamelCase(facetDataArray[0].columnName, facetAliases))
     return (
       <div className="FacetPlotsCard cardContainer">
-        <div className="FacetPlotsCard__titlebar" style={{backgroundColor: colorPalette[0].replace(')', ',.05)')}}>
-          <span className="FacetPlotsCard__title">
-            {cardTitle}
-          </span>
+        <div
+          className="FacetPlotsCard__titlebar"
+          style={{ backgroundColor: colorPalette[0].replace(')', ',.05)') }}
+        >
+          <span className="FacetPlotsCard__title">{cardTitle}</span>
           {isLoading && (
             <span style={{ marginLeft: '2px' }} className={'spinner'} />
           )}
         </div>
         <div className="FacetPlotsCard__body">
           {/* create a plot for every facet to be plotted */}
           {facetPlotDataArray.map((plotData, index) => {
-            return <div>
-              {index !== 0 && <hr></hr>}
-              {isShowingMultiplePlots && 
-                <div className="FacetPlotsCard__body__facetname">
-                  <span>
-                      {unCamelCase(facetDataArray[index].columnName, facetAliases)}
-                  </span>
-                </div>
-              }
-              <div className="FacetPlotsCard__body__row">
-                <SizeMe monitorHeight>
-                  {({ size }) => (
-                    <div className="FacetPlotsCard__body__plot">
-                      <Plot
-                        key={`${facetsToPlot![index]}-${size.width}`}
-                        layout={layout}                      
-                        data={plotData?.data ?? []}
-                        style={getPlotStyle(
-                          size.width,
-                          'PIE',
-                          150,
-                        )}
-                        config={{ displayModeBar: false }}                  
-                      />
-                    </div>
-                  )}
-                </SizeMe>
-                {renderLegend(plotData?.labels, plotData?.colors, false)}
+            return (
+              <div key={index}>
```

Comment:
Linting error that this div needed a key. Remainder of changes (I think) are caused by code formatter.